Makefile

```diff
@@ -128,7 +128,7 @@ dc:
 		PYTHON_VERSION=$${PYTHON_VERSION:-$(DEFAULT_PYTHON_VERSION)} \
 		RAPIDS_NAMESPACE=$${RAPIDS_NAMESPACE:-$(DEFAULT_RAPIDS_NAMESPACE)} \
 		CONDA_CUDA_TOOLKIT_VERSION=$${CONDA_CUDA_TOOLKIT_VERSION:-$$CUDA_VERSION} \
-		docker-compose -f $(file) $(cmd) $(cmd_args) $(svc) $(svc_args)
+		docker compose -f $(file) $(cmd) $(cmd_args) $(svc) $(svc_args)
```

Comment:
This is technically only valid if the user has the docker-compose v2 plugin. I'd like to keep it as `docker-compose` to support v1, because users with v2 can install the [`compose-switch`](https://github.com/docker/compose-switch) plugin.